db/seeds.rb

```diff
@@ -1,8 +1,455 @@
+                            year: 2021,
+                            name: '9º Ano',
+                            starts_on: Date.new(2021, 1, 1),
+                            ends_on: Date.new(2021, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '5º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '6º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '7º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '8º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2022,
+                            name: '9º Ano',
+                            starts_on: Date.new(2022, 1, 1),
+                            ends_on: Date.new(2022, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '5º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '6º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '7º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '8º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          },
+                          {
+                            year: 2023,
+                            name: '9º Ano',
+                            starts_on: Date.new(2023, 1, 1),
+                            ends_on: Date.new(2023, 12, 31)
+                          }])
+
+GRADES = ['5º Ano', '6º Ano', '7º Ano', '8º Ano', '9º Ano'].freeze
```

Comment:
You even created this variable, it could have been used up there, in the courses variable.